.changeset/icy-results-wonder.md

```diff
@@ -0,0 +1,19 @@
+---
+"@biomejs/biome": patch
+---
+
+Fixed [#6569](https://github.com/biomejs/biome/issues/6569): Allow files to export from themselves with `noImportCycles`.
+
+This means the following is now allowed:
+
+**example.js**
+```js
+export function example1() {
+  return 1;
+}
+
+export function example2() {
+  return 2;
+}
+
+export * as Example from './test';
```

Comment:
The example is not very clear, can you please:
- write a small paragraph that **explains the example** e.g. from the file example.js you can...